### DISCUSSION TITLE
Have you all ever thought about using material-you which is the latest material-design version?

### DISCUSSION THREAD

**Discussion by ghost** [fyne-io/fyne#3260] [Q&A] [answered]
Hi all! I would like to know if everyone thought of using material-you which is the most recent and current material-design version today.

**It's more of a question than a request for an appeal.**

**Comment by andydotxyz** [comment, marked as answer]
We started with material design.
For 2.3 we have updated it to our own style which is similar but refined for a better platform agnostic UI.
